engineapi/backup_monitor.go

```diff
@@ -109,6 +123,68 @@ func NewBackupMonitor(logger logrus.FieldLogger,
 	return m, nil
 }
 
+// getBackupVolumeRecurringJobs get recurring jobs in the volume labels and recurring jobs of groups in the volume labels
+func (m *BackupMonitor) getBackupVolumeRecurringJobs(ds *datastore.DataStore, volume *longhorn.Volume) (string, error) {
+	allRecurringJobs, err := ds.ListRecurringJobsRO()
+	if err != nil {
+		m.logger.WithError(err).Warn("Failed to list all recurring jobs")
+		return "", err
+	}
+
+	volumeRecurringJobSpecList := make(map[string]longhorn.RecordVolumeRecurringJob)
```

Comment:
The naming can be better, so it will be easy to connect to the method purpose.

```go
volumeRecurringJobInfos := make(map[string]longhorn.RecordVolumeRecurringInfo)
```

Reply:
Modified and all related codes. Thanks